bemoji

```diff
@@ -31,6 +33,7 @@ usage() {
     echo "   -e             Only echo out the picked emoji."
     echo ""
     echo " Other options:"
+    echo "   -n             No newline after the picked emoji."
```

Comment:
Can you change this to 'Do not print a newline character after the picked emoji.' so it fits with the other descriptions?